### PR TITLE
Fixes #24520 - set default number of RSS posts

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Review by ik5 — DISMISSED** [review]
LGTM

File: lib/tasks/rss.rake [hunk 1]
```diff
@@ -31,9 +31,12 @@ Examples:
 END_DESC
 
   task :create_notifications => :environment do
+    posts = ENV['FOREMAN_RSS_LATEST_POSTS'].to_i
+    posts = 3 if posts == 0
```
Comment (tbrisker) [inline lib/tasks/rss.rake]: Wouldn't it be better to set the default only if there is no ENV variable, allowing setting it to 0 to disable?

Comment (ares) [reply lib/tasks/rss.rake]: updated

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Pull request merged**